### PR TITLE
Bump aidbox&multibox versions

### PR DESCRIPTION
**Pull request by MFAshby**

### CODE REVIEW

File: scripts/docker-compose.yaml [hunk 1]
```diff
@@ -40,8 +40,8 @@ services:
       POSTGRES_DB:       "postgres"
 
   multibox:
-    # Version pinned 2022-06-18 tag healthsamurai/multibox:edge
-    image: "healthsamurai/multibox@sha256:9492b7a6e92b04b3389c52116e68c7550699162f5550b454e1f3d11cb008627b"
+    # Version pinned 2022-10-18 tag healthsamurai/multibox:edge
+    image: "healthsamurai/multibox@sha256:a621c50812578434d0d3af29733da9687ac053699c3a770bfad9b81e993dcc87"
```
Comment (idasbiste) [inline scripts/docker-compose.yaml:44]: There is a new `edge` version pushed yesterday. Hash `a19b9d446cfce4e067d1107a0d3a206f9827d0517dcdef9e3f2820ad8cfef6b2`

File: scripts/docker-compose.yaml [hunk 2]
```diff
@@ -11,8 +11,8 @@ services:
       POSTGRES_DB:       "postgres"
 
   devbox:
-    # :latest, pinned on 2022-08-27
-    image: "healthsamurai/devbox@sha256:a010d77ce2fa31222ae334fabc529734d6fbf5aa09be8bc2c2f88f2fe8aa707b"
+    # :edge, pinned on 2022-10-19
+    image: "healthsamurai/devbox@sha256:4bab6ec177883886d6b10875737dc11083be123a6d16565c30a0324881477fa4"
```
Comment (idasbiste) [inline scripts/docker-compose.yaml:15]: There is a new `edge` version pushed yesterday. Hash `ee2a687d25a73dd978bb0db4fdc5e116656dcb5b56ec1868f69620676a3a1564`

**Review by idasbiste — CHANGES_REQUESTED** [review]
@MFAshby I _think_ I was seeing it correctly but the sha256 digests are different from what you have here for both cases.

File: scripts/docker-compose.yaml (hunk 1 above)
Comment (MFAshby) [reply scripts/docker-compose.yaml:44]: haha I think they push a new one every day via their CI...

Comment (MFAshby) [reply scripts/docker-compose.yaml:44]: I will upgrade tho

Comment (MFAshby) [reply scripts/docker-compose.yaml]: Fixed.

File: scripts/docker-compose.yaml (hunk 2 above)
Comment (MFAshby) [reply scripts/docker-compose.yaml]: Fixed

File: scripts/docker-compose.yaml [hunk 3]
```diff
@@ -11,8 +11,8 @@ services:
       POSTGRES_DB:       "postgres"
 
   devbox:
-    # :latest, pinned on 2022-08-27
-    image: "healthsamurai/devbox@sha256:a010d77ce2fa31222ae334fabc529734d6fbf5aa09be8bc2c2f88f2fe8aa707b"
+    # healthsamurai/aidboxone:edge, pinned on 2022-10-20
+    image: "healthsamurai/aidboxone@sha256:dc25aa007dab5570317d812aeff3ca5a6b055b7300b5b66e6d6cdaa6d97f0f7f"
```
Comment (k9t5h) [inline scripts/docker-compose.yaml:15]: Why did you go with `aidboxone` instead of `devbox`?

Comment (MFAshby) [reply scripts/docker-compose.yaml:15]: to follow their documentation; it no longer mentions `devbox` image so I assume they are phasing it out 
https://docs.aidbox.app/getting-started/run-aidbox-locally-with-docker#create-.env-file

File: scripts/docker-compose.yaml [hunk 4]
```diff
@@ -40,8 +40,8 @@ services:
       POSTGRES_DB:       "postgres"
 
   multibox:
-    # Version pinned 2022-06-18 tag healthsamurai/multibox:edge
-    image: "healthsamurai/multibox@sha256:9492b7a6e92b04b3389c52116e68c7550699162f5550b454e1f3d11cb008627b"
+    # healthsamurai/multibox:edge, pinned 2022-10-20
+    image: "healthsamurai/multibox@sha256:da8c348c00a7d2121521ed058e70b9fd366ffad1a016856f9e9e17eab1d7cc9c"
```
Comment (k9t5h) [inline scripts/docker-compose.yaml:44]: Where does this come from?
I have only found `sha256:a19b9d446cfce4e067d1107a0d3a206f9827d0517dcdef9e3f2820ad8cfef6b2` on the following link:
https://hub.docker.com/layers/healthsamurai/multibox/edge/images/sha256-a19b9d446cfce4e067d1107a0d3a206f9827d0517dcdef9e3f2820ad8cfef6b2?context=explore

Comment (MFAshby) [reply scripts/docker-compose.yaml:44]: from running `docker pull healthsamurai:edge`, about 20 minutes ago

Comment (k9t5h) [reply scripts/docker-compose.yaml:44]: I'm too UI oriented, thanks:D

Comment (MFAshby) [reply scripts/docker-compose.yaml:44]: no worries, I don't know why the docker hub UI shows another hash :(

**Review by k9t5h — APPROVED** [review]

**Review by idasbiste — APPROVED** [review]

**Pull request merged**